### PR TITLE
Obtain S/MIME signature using cryptography library

### PR DESCRIPTION
**Pull request by kyrofa**
The governance and permission files must be S/MIME signed. The biggest obstacle to adopting the cryptography library is that it doesn't expose an S/MIME API (see https://github.com/pyca/cryptography/issues/1621 for more information).
    
This PR unblocks this and makes progress on #109 by using the OpenSSL API directly (via cryptography's hazard layer) to obtain S/MIME signatures, and prove the functionality out by using it to sign the governance files.

### CODE REVIEW

**Comment by emersonknapp** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/7397/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/3554/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/6071/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/7226/)

File: sros2/sros2/api/__init__.py [hunk 1]
```diff
@@ -21,6 +21,10 @@
 import sys
 
 from lxml import etree
```
Comment (jacobperron) [inline sros2/sros2/api/__init__.py]: linter error :smile:
The cryptography imports should come before the lxml import.

Comment (kyrofa) [reply sros2/sros2/api/__init__.py]: Whoops, sorry-- fixed!

**Comment by kyrofa** [comment]
@jacobperron those macOS failures look unrelated, any chance that's the wrong link?

**Comment by jacobperron** [comment]
Yep, that was the wrong link. Updated.

**Pull request merged**